roles/unbind-unifiedpush-apb/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- debug:
+    msg: "Sync does not need to release resources on unbind.."
```

Comment:
Is this some role for testing?

Reply:
Unbind failed if I didn't have an unbind role.  I suppose I could have looked more into it, but it seemed better to put something that was of the "page intentionally left blank" ilk

Reply:
Ok I just double checked, I 100% committed this director by accident.